akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1356,7 +1356,7 @@ trait FlowOps[+Out, +Mat] {
    * '''Cancels when''' downstream cancels
    */
   def throttle(elements: Int, per: FiniteDuration, maximumBurst: Int, mode: ThrottleMode): Repr[Out] =
-    throttle(elements, per, maximumBurst, _ ⇒ 1, mode)
+    throttle(elements, per, maximumBurst, ConstantFun.oneInt, mode)
```

Comment:
This is in that raw form is not very informative to the user. Let's add a human digestible sentence as well.